src/styles/partials/components/_site-header.scss

```diff
@@ -39,20 +39,34 @@
 }
 
 @media screen and (max-width: $header-breakpoint) {
+  #search-form {
+    display: none;
+    padding: 15px;
+    position: absolute;
+    top: 85px;
+    height: 65px;
+    width: 100%;
+    background: white;
+    z-index: 999999999;
+    flex-direction: column;
+    justify-content: center;
+  }
+
   .dg_site-header {
     background: $white;
     position: relative;
     z-index: 99;
     margin-bottom: 64px; /* Visually calculated for this component only */
   }
 
-  .dg_search-container {
-    display: none;
+  #search-form.is-visible {
+    display: flex;
   }
 
   .dg_mobile-search-button {
     display: block;
     padding: $default-padding;
+    z-index: $max-z-index-minus-1;
```

Comment:
Is the z-index property necessary? The button does not appear to change in an adverse way when I remove it.

Reply:
@theilman7 if its not there, you cannot select the search icon on smaller screens. At least taht is what happened for me, can you test your change as see if thats consistent for you?

Reply:
these shouldn't be in this pr